libqtile/xcbq.py

```diff
@@ -850,8 +887,18 @@ def __init__(self, display):
                 setattr(self, i, self._extmap[i](self))
 
         self.pseudoscreens = []
-        if "xinerama" in extensions:
-            for i, s in enumerate(self.xinerama.query_screens()):
```

Comment:
Here also.

Reply:
Here, the idea was that Xinerama is less used nowadays than RandR so it makes sense to only test Xinerama if "randr" is not in extensions. 
But it's details, should I rollback that change?

Reply:
I thought the diff was just about the quotes style, I didn't see the `elif` being added. That makes it a good point to explain why we should only do those style changes in separated commits/PR! :stuck_out_tongue: